packages/frontend-2/components/projects/Dashboard.vue

```diff
@@ -18,41 +18,36 @@
     </div>
 
     <PromoBannersWrapper v-if="promoBanners.length" :banners="promoBanners" />
-    <div
-      v-if="!showEmptyState"
-      class="flex flex-col space-y-2 md:flex-row md:items-center mb-8 pt-4"
-    >
-      <h1 class="text-heading-xl">Projects</h1>
-
-      <div
-        class="flex flex-col space-y-2 sm:space-y-0 sm:flex-row sm:items-center sm:space-x-2 grow md:justify-end"
-      >
-        <FormTextInput
-          v-model="search"
-          name="modelsearch"
-          :show-label="false"
-          placeholder="Search projects..."
-          color="foundation"
-          wrapper-classes="grow md:grow-0 md:w-60"
-          :show-clear="!!search"
-          @change="updateSearchImmediately"
-          @update:model-value="updateDebouncedSearch"
-        ></FormTextInput>
-        <div class="flex items-center space-x-2">
+
+    <div v-if="!showEmptyState" class="flex flex-col gap-4">
+      <div class="flex items-center gap-2 mb-2">
+        <Squares2X2Icon class="h-5 w-5" />
+        <h1 class="text-heading-lg">Projects</h1>
+      </div>
+
+      <div class="flex flex-col sm:flex-row gap-2 sm:items-center justify-between">
+        <div class="flex flex-col sm:flex-row gap-2">
+          <FormTextInput
+            name="modelsearch"
+            :show-label="false"
+            placeholder="Search..."
+            :custom-icon="MagnifyingGlassIcon"
+            color="foundation"
+            wrapper-classes="grow md:grow-0 md:w-60"
+            :show-clear="!!search"
+            :model-value="bind.modelValue"
```

Comment:
`v-bind="bind"` w/ the latest changes to useDebouncedTextInput (assuming you have main merged in) - change this everywhere